src/openrct2/paint/tile_element/Paint.LargeScenery.cpp

```diff
@@ -388,14 +386,11 @@ void PaintLargeScenery(PaintSession& session, uint8_t direction, uint16_t height
         }
     }
 
-    auto boxlengthZ = std::min<uint8_t>(tile->z_clearance, 128) - 3;
-    auto flags = tile->flags;
+    auto boxlengthZ = std::min(tile.zClearance, 128) - 3;
     auto bbIndex = 16;
-    if (flags & 0xF00)
+    if (tile.walls) // ODD
```

Comment:
What does this comment mean? That is just fires on odd edges or that the code is odd?

Reply:
i forgot to delete it...
Its odd code that it checks for if there are walls but then only operates on the corners variable.

Reply:
![image](https://github.com/user-attachments/assets/9c2f7a81-2943-4b46-a013-6a898ad782c8)
confirmed that its like that in vanilla at least

Reply:
I've put in a better comment with a reason as to why the odd code makes sense.